mautic/mautic
# TRPOD 312 - TPROD 358 - CoreBundle Theme: migrate views from php engine to twig

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
- migrated template(s) responsible for the Themes overview (/s/themes)
- in that process, I also had to tweak some related templates from Default (eg. content.html.twig, base.html.twig & slim.html.twig, …).
- Making these changes, meant switching to the twig templates instead of php templates for Left panel, Right panel, global search and notifications panel on some pages that did not use them yet.
- The impact is on pages that use the same 'list' component as the Themes overview (eg. Segments overview)
- Also impacts their detail pages (the View & Edit), as these inherit the same base templates to load the global search and the actions (buttons on top right).



#### Steps to test this PR:
- go to /s/themes:
  - you should see the list of themes, with title, actions, … same as before
  - you can still use the actions to download or preview
  - left & right sidebar should still show menu items and work when clicked
  -  clicking search icon in header should still open up search and give you the expected results
  - notifications icon should still open up notifications pane
- other first-level navigation items that contain lists, should also still show their list and you should be able to click through to detail. These should also have working sidebars, search and notifications pane.
For example:
  - /s/contacts
  - /s/companies
  - /s/segments
  - s/forms
  - s/pages
- for good measure, you should also check the detail and Edit view of one of the contacts, segments, companies. This to make sure the buttons display and work, search works, sidebars work


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.
